connexion/apis/aiohttp_api.py

```diff
@@ -208,8 +208,12 @@ def get_response(cls, response, mimetype=None, request=None):
         if isinstance(response, ConnexionResponse):
             response = cls._get_aiohttp_response_from_connexion(response, mimetype)
 
-        logger.debug('Got data and status code (%d)',
-                     response.status, extra={'data': response.body, 'url': url})
+        if isinstance(response, web.StreamResponse):
```

Comment:
Can you make this an `elif`?

Reply:
I'm assuming that `ConnexionResponse` never wraps `web.StreamResponse`. If I change this to elif there's additional debug print that was not there before.

Is this intentional?

What are possibilities here by the way?

- `web.Response`
- `web.StreamResponse`
- `ConnexionResponse`

anything else?

Reply:
@jmcs can you please look into this? I want to know how to proceed.

Reply:
NVM. You're right.